docs/en/04-supported-codebases/mmcls.md

```diff
@@ -1,20 +1,149 @@
-# MMClassification Support
+# MMClassification Deployment
 
-[MMClassification](https://github.com/open-mmlab/mmclassification) is an open-source image classification toolbox based on PyTorch. It is a part of the [OpenMMLab](https://openmmlab.com) project.
+[MMClassification](https://github.com/open-mmlab/mmclassification) aka `mmcls` is an open-source image classification toolbox based on PyTorch. It is a part of the [OpenMMLab](https://openmmlab.com) project.
 
-## MMClassification installation tutorial
+## Installation
 
-Please refer to [install.md](https://github.com/open-mmlab/mmclassification/blob/master/docs/en/install.md) for installation.
+### Install mmcls
 
-## List of MMClassification models supported by MMDeploy
+If you have already done that, move on to the next section. Otherwise, please follow this [quick guide](https://github.com/open-mmlab/mmclassification/tree/1.x#installation) to finish mmcls installation.
```

Comment:
add link: 
.... to the next section. ---> to the [next section](#install-mmdeploy)

Reply:
Done